### PR TITLE
Added environment variables and Paybill APIs

### PR DESCRIPTION
**Pull request by truthfool**
- Added Paybill apis in payment hub collection
- Added necessary environment variables in SIT environment json
Note : Paygops does not have ingress defined so kept it as localhost.
@fynmanoj

### CODE REVIEW

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: PostmanCollections/Payment Hub.json [hunk 1]
```diff
@@ -44798,6 +44798,754 @@
 					"response": []
 				}
 			]
+		},
+		{
+			"name": "Paybill APIs",
```
[the remark this answers is not in the document]
Comment (truthfool) [reply PostmanCollections/Payment Hub.json:44803]: Ok got it.

File: PostmanCollections/Environment/SIT.json [hunk 2]
```diff
@@ -174,9 +174,21 @@
...
+		},
+		{
+			"key": "amsName",
+			"value": "paygops",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "PaygopsHostName",
+			"value": "http://0.0.0.0:5002",
```
[the remark this answers is not in the document]
Comment (truthfool) [reply PostmanCollections/Environment/SIT.json:186]: I will create ingress for paygops and roster both and add it.

File: PostmanCollections/Environment/SIT.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-	"id": "2e96d191-bdf9-431c-9b7b-417895cc08a5",
+	"id": "3ce2e568-86c7-4f17-a261-94a33bd2951b",
 	"name": "SIT",
```
[the remark this answers is not in the document]
Comment (truthfool) [reply PostmanCollections/Environment/SIT.json:3]: I will keep the id as previous such that the integration tests will run.

[the remark this answers is not in the document]
Comment (fynmanoj) [reply PostmanCollections/Environment/SIT.json:3]: integration test is not run using the ID, It is run based on the name in one environment variable

**Pull request closed without merging**